src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.scss

```diff
@@ -167,3 +167,12 @@ label,
   margin: 3px;
   cursor: pointer;
 }
+
+.address {
+  width: 500px;
```

Comment:
will this work for mobile ?

Reply:
**This screenshot was captured on iPhone X/XS width (375px)**


![231](https://user-images.githubusercontent.com/46484914/120383463-9246f300-c32d-11eb-93b4-df43d1680cd8.jpg)